sass/directives/_boxes.scss

```diff
@@ -10,3 +9,23 @@
   padding: $base-padding;
   background-color: $panel-primary;
 }
+
+// Accordion Panels
```

Comment:
For future reference, I'd split this new functionality into its own file - i.e., `directives/_accordions.scss`. I can do this myself when I refactor this lib.